why_angular_2.md

```diff
@@ -1,38 +1,52 @@
+With many front-end JavaScript frameworks to choose from, it's natural to wonder what makes Angular stand apart from the rest. Coming from a history of wide community adoption and engagement with AngularJS, Angular improved on the foundations to make the framework faster, more scalable, and more modern.
+
+Organizations that found value in AngularJS will find more value in Angular. It is increasingly common to see sentiment in the enterprise space that Angular provides the sort of 'out of box' functionality and opinionated tools that allow for large teams to work cohesively and consistently.
 
 ## Angular's Advantages
 
-The first release of AngularJS provided programmers with the tools to develop and architect large scale JavaScript applications, but its age has revealed a number of flaws and sharp edges. Angular was built on five years of community feedback.
+The first release of AngularJS provided programmers with the tools to develop and architect large-scale JavaScript applications, but age has revealed some flaws and sharp edges. Angular was built on top of the five years of community feedback from the AngularJS days.
 
-Now, after the many years of Angular 2+ released, it has matured into a system focused on providing tools that scale, with a robust community and many common problems solved with tutorials, blog posts, and videos galore to help you tackle challenges of all shapes and sizes.
+Angular has matured into a system focused on providing tools that scale, with a robust community and many common problems solved with tutorials, blog posts, and videos galore to help you tackle challenges of all shapes and sizes.
 
-### Angular Is Easier
+### Easy to Learn, Difficult to Master
 
-The new Angular codebase is more modern, more capable and easier for new programmers to learn than AngularJS, while also being easier for project veterans to work with.
+One of Angular's biggest advantages is how its structure supports learning and understanding. Developers new to the framework can tackle independent aspects of the framework, build an understanding, and expand their understanding to the next concept when they're ready. Teams that keep their code structure and guidelines close to the [Angular Styleguide](https://angular.io/guide/styleguide) will see that new developers can more quickly spin up and work on complex engineering problems using this base documentation to accelerate their understanding of your codebase.
```

Comment:
> Developers new to the framework can tackle independent aspects of the framework, build an understanding, and expand their understanding to the next concept when they're ready.

The next concept? This is speaking to a progression but it's hard for me to understand. Are we trying to say that you can learn a subset of the framework's features to solve a problem and later leverage that understanding to understand more advanced features?  All this talk of systems, concepts and tackling aspects of the framework is vague.